website/docs/reference/resource-properties/description.md

```diff
@@ -336,15 +343,11 @@ models:
 
 </File>
 
-_[CLI users only]_
-
 3. Run `dbt docs generate` — the `assets` directory will be copied to the `target` directory
 
 4. Run `dbt docs serve` — the image will be rendered as part of your project documentation:
 
-<Lightbox src="/img/reference/image-in-docs.png" title="The image at assets/dbt-logo.svg is rendered correctly"/>
-
-If mixing images and text together, also consider using a docs block.
+If mixing images and text, also consider using a docs block.
 
 ### Include an image from the web in your descriptions
```

Comment:
@joellabes do you know whether both cloud and core users can do this? or only cloud users can incl an image from the web?

Reply:
If anything, it'll be the opposite and cloud users won't be able to use images from the web due to CORS restrictions. [Discussed here](https://dbt-labs.slack.com/archives/C04HJR9B69H/p1705017810057809?thread_ts=1705017455.550139&cid=C04HJR9B69H)

Reply:
ah ok, but for explorer right @joellabes ? for dbt docs, it'll still be the same? i thought this doc was for dbt docs per the comment [above](https://github.com/dbt-labs/docs.getdbt.com/pull/4756/files#r1455257657) but now realize it's for both explorer and dbt docs!